docs/xxjitserveraotcacheignorelocalscc.md

```diff
@@ -0,0 +1,50 @@
+﻿<!--
+* Copyright (c) 2017, 2024 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# -XX:[+|-]JITServerAOTCacheIgnoreLocalSCC
+
+This option, which is used at JITServer client VMs, controls how the JITServer AOT cache feature interacts with the local shared classes cache. You can enable or disable storing and loading of JITServer AOT cache without a local shared classes cache.
```

Comment:
"You can enable or disable storing and loading of JITServer AOT cache"  is not technically correct because it's not the AOT cache that gets stored or loaded, but rather artifacts from the AOT cache. Maybe more precise is: "You can enable or disable the usage of JITServer AOT cache artifacts (AOT methods and associated metadata) in the absence of a local shared class cache with write permissions and free space.